install_config/redeploying_certificates.adoc

```diff
@@ -688,6 +682,24 @@ $ oc create secret tls router-certs --cert=router.pem \ <1>
 <1> *_router.pem_* is the file that contains the concatenation of the
 certificates that you generated.    
 
+. Redeploy the router:
++
+----
+$ oc rollout latest dc/router
+----
+
+When routers are initially deployed, an annotation is added to the router's
+service that automatically creates a
+xref:../dev_guide/secrets.adoc#service-serving-certificate-secrets[service serving certificate secret] named `router-metrics-tls`.
+
+To redeploy `router-metrics-tls` certificates manually, that service serving certificate can be triggered to be recreated by deleting the secret, removing and re-adding annotations to the router service, then redeploying the `router-metrics-tls` secret:
```

Comment:
... can be triggered to be recreated by `removing annotations to the router service,  deleting the secret and re-adding annotations to the router service`, I think the secret should be deleted after removing the annotation, otherwise it will be recreated immediately.

Reply:
Line 708: `service.alpha.openshift.io/serving-cert-signed-by-` this annotation should not be removed.

Reply:
others LGTM, thanks

Reply:
Thank you for your review and kind correction @lihongan,

You're right. I changed the sequence of the steps as you mentioned. (removing secret step is after removing annotation step) 

And I think it had better to remove `service.alpha.openshift.io/serving-cert-signed-by` annotation at same time. The annotation also added automatically with the current `OpenShift CA` signer value, when it is added `service.alpha.openshift.io/serving-cert-secret-name` again. For example, if new `CA` is deployed before the `router certs` deploying work, it can cause not matching CA issue with new `CA`, because the `service.alpha.openshift.io/serving-cert-signed-by`'s value is old or different with new `CA` signer value.

Reply:
That make sense. Thank you @bysnupy